hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/LeaseManager.java

```diff
@@ -583,11 +583,7 @@ public void acquireLock(TransactionLocks locks) throws IOException {
                 .add(lf.getLeaseLockAllPaths(LockType.WRITE, holder,
                         fsnamesystem.getLeaseCreationLockRows()))
                 .add(lf.getLeasePathLock(leasePaths.size()))
-                .add(lf.getBlockLock()).add(lf.getBlockRelated(BLK.RE, BLK.CR, BLK.IV, BLK.ER,
-                    BLK.UC, BLK.PE, BLK.UR));
-            if (fsnamesystem.isCloudEnabled()) {
-              locks.add(lf.getProvidedCacheLocLoc());
-            }
+                .add(lf.getBlockLock()).add(lf.getBlockRelated(BLK.RE, BLK.CR, BLK.ER, BLK.UC, BLK.UR));
```

Comment:
actually it seems to be missing BLK.IV and BLK.PE. Is that intentional?
```
BLK.RE, BLK.CR, BLK.IV, BLK.ER,
                    BLK.UC, BLK.PE, BLK.UR
```

Reply:
compare it with this https://github.com/hopshadoop/hops/commit/2a07908f60944ac714b033e96d4bdfbe44ea88cf

Reply:
IV and PE are from the enterprise branch that got added by mistake

Reply:
aha, got it, thanks!